CMakeLists.txt

```diff
@@ -30,9 +30,18 @@ macro(clangformat_targets)
     message(WARNING "Not generating clangformat target, no clang-format tool found")
   else ()
     message(STATUS "Generating clangformat target using ${CLANG_FORMAT}")
-    file(GLOB_RECURSE ALL_SOURCE_FILES src/*.cc src/*.h src/*.hh)
+    find_package(Git)
+    execute_process(
+      COMMAND ${GIT_EXECUTABLE} ls-files *.cc *.h *.hh
```

Comment:
NIT: I didn't find any .hh file in the source...

Reply:
We did, but not since we went public.  I can remove it.

Also, do we want clangformat to work on the 'td' files?

Reply:
Last time I tried to run clangformat on the td files it worked very badly.

Reply:
Ignore td files. :)